projects/plugins/jetpack/extensions/shared/components/inspector-notice/index.jsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import { JetpackLogo } from '@automattic/jetpack-components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export default function InspectorNotice( { children } ) {
+	return (
+		<div className="jetpack-inspector-notice">
+			<span>{ children }</span>
+			<JetpackLogo height="16" logoColor="#1a1a1a" showText={ false } />
```

Comment:
The tests are failing because the height should be a number:

```suggestion
			<JetpackLogo height={ 16 } logoColor="#1a1a1a" showText={ false } />
```